etc/docs/README.md

```diff
@@ -0,0 +1,54 @@
+# docs_utils
+
+This directory contains scripts to generate api docs as well our the Hugo site template used for the MongoDB node driver documentation.
+
+There are two scripts contained in this folder.
+
+- `legacy-generate.sh` was used to generate API documentation before the driver's docs
+were moved into the main repository.  This script has the ability to generate api docs for older versions of the driver (in case it becomes
+necessary to backport a feature).
+
+- `generate-docs.ts` is used to generate API docs for a major or minor release.
+
+### Dependencies
+
+`generate-docs.ts` requires the following in addition to dependencies installed with `npm i`:
+
+* Hugo static web generator `v0.30.2`
+  * You can download the right version [here](https://github.com/gohugoio/hugo/releases/tag/v0.30.2)
+* typedoc 
```

Comment:
this was added to dev dependencies, will it need to be installed separately, too, or should it not be in dev dependencies?